src/main/java/cms/gov/madie/measure/services/BaseMeasureService.java

```diff
@@ -139,4 +161,100 @@ public void verifyAuthorization(String username, Measure measure, List<RoleEnum>
     MeasureServiceUtil.verifyMeasureSetAuthorization(
         username, "Measure", measure.getId(), roles, measureSet);
   }
+
+  void extractUserDeets(Map<String, UserDetailsDto> userDetailsMap, MeasureListDTO measure) {
```

Comment:
Let's stick with common English and clear abbreviations.
```suggestion
  void extractUserDetails(Map<String, UserDetailsDto> userDetailsMap, MeasureListDTO measure) {
```